Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,124 @@
+```
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/>
+      </startup>
+      <runtime>
+        <!-- AppContextSwitchOverrides value attribute is in the form of 'key1=true|false;key2=true|false  -->
+        <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false" />
+      </runtime>
+    </configuration>
+```
+pplications that target the .NET Framework 4.7.1 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to ```true```.
+
+For an overview of UI automation, see the [UI Automation Overview](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview).
+
+__Added support for UI Automation patterns and properties__
+
+The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns and properties:
+- The `T:System.Windows.Forms.ToolStripSplitButton` and `T:System.Windows.Forms.ComboBox` controls support the [Expand/Collapse pattern](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern)
+- The `T:System.Windows.Forms.ToolStripMenuItem` control supports the [ControlType](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value "MenuItem"
+- The `T:System.Windows.Forms.ToolStripItem` controls supports the [Name](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationelement.nameproperty?view=netframework-4.7) property and [Expand/Collapse pattern](https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern)
+- [AccessibleEvents](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.accessibleevents?view=netframework-4.7) indicating StateChange and NameChange when drop down is expanded or collapsed to `T:System.Windows.Forms.ToolStripDropDownItem`
+- The `T:System.Windows.Forms.ToolStripDropDownButton` control property has a ControlType property value of `F:System.Windows.Automation.ControlType.Button` 
+- `T:System.Windows.Forms.DataGridViewCheckBoxCell` control supports the [Toggle Pattern](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.togglepattern?view=netframework-4.7)
+ 
+__Improvements to PropertyBrowser control__
+
+The .NET Framework 4.7.1 adds the following improvements to the PropertyBrowser control: 
+- The __Details__ button in the error dialog that is displayed when the user enters an incorrect value in the T:System.Windows.Forms.PropertyGrid control supports the Expand/Collapse pattern, state and name change notifications, and a ControlType property with a value of "Button".
```

Comment:
Can "ControlType property with a value of 'Button'" be expressed as an xref to ControlType.Field?